BankApplication/src/com/luxoft/bankapp/comands/TransferCommand.java

```diff
@@ -0,0 +1,43 @@
+package com.luxoft.bankapp.comands;
+
+import java.util.Scanner;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.luxoft.bankapp.model.Client;
+import com.luxoft.bankapp.service.BankService;
+import com.luxoft.bankapp.service.BankServiceImpl;
+
+
+
+public class TransferCommand implements Command {
+
+	private final static Logger LOG = LoggerFactory.getLogger(TransferCommand.class);
+	
+	@Override
+	public void execute() {
+		Scanner scan=new Scanner(System.in);
```

Comment:
You have bunch of scanners everywhere. I think here is a good place to actually use singletone pattern and create separate IO class, so you won't be creating scanners everywhere.